clang/lib/CIR/CodeGen/CIRGenCall.cpp

```diff
@@ -889,6 +900,18 @@ RValue CIRGenFunction::emitCall(const CIRGenFunctionInfo &CallInfo,
           auto Results = theCall->getOpResults();
           assert(Results.size() <= 1 && "multiple returns NYI");
           assert(Results[0].getType() == RetCIRTy && "Bitcast support NYI");
+
+          auto region = builder.getBlock()->getParent();
```

Comment:
In the past few months our coding guidelines now asks for explicit types for most things (we'll need to use them for upstreaming), so going forward you need to fix these, the one relevant here:
`mlir::Region *region = builder.getBlock()->getParent();`